lxd/apparmor.go

```diff
@@ -317,12 +370,13 @@ func runApparmor(command string, c container) error {
 		return nil
 	}
 
-	cmd := exec.Command("apparmor_parser", []string{
+	args := []string{
 		fmt.Sprintf("-%sWL", command),
 		path.Join(aaPath, "cache"),
 		path.Join(aaPath, "profiles", AAProfileShort(c)),
-	}...)
+	}
 
+	cmd := exec.Command("apparmor_parser", args...)
```

Comment:
It probably is. I suppose it's left over from some previous incantation of how I was doing this.